### PR TITLE
Small Code Improvements for Review

### PR DESCRIPTION
**Pull request by RahulTeammate**
Hello Guad,

I've gone through the codebase and understand the basic principle of what's happening. While doing so, I found a few areas for improvement:

1. In Characters,cs, the byteWritten variables in the setMessage and setCharacter functions are unused. I commented them out in my fork, but are these variables going to be used in a future online feature? If not, we can omit these lines from the code.
2. Also in Characters.cs, we reverse an array of 4 0s. Removing this redundant line will improve online speeds, especially when the game is run through Hostinger instead of someone's IP addr.
3. In Form1.cs, there is a double initialization of the static members playerClient, connection, and listener. I commented out their initializations in the constructor because each new initialization of Form1.cs would reset the 3 static variables. If we want these variables to be independent to each instance of Form1, we need to remove their "static" keyword as well.
4. In Program.cs line 67, we do not log or show a socket exception from a dns lookup fails. I added one for error tracking purposes. If we should not respond to a socket exception from Program.cs, I can remove this line.
5. In Form1.cs, lines 836 and 839, I removed the redundant "+ 0" operations. This change can improve compiler optimizations + .Net interpreter speed.

I have ran the project on VS 2017 and analyzed the stack trace/ output. Nothing out of the ordinary as occurred in response to my changes.

### CODE REVIEW

File: SM64O/Program.cs [hunk 1]
```diff
@@ -66,6 +66,7 @@ public static IPAddress ResolveAddress(string input)
                 }
                 catch (SocketException)
                 {
+                    Program.LogException(new SocketException());
```
Comment (Guad) [inline SM64O/Program.cs]: we dont need this

**Comment by RahulTeammate** [comment]
Ok, Guad. I'll remove that. Anything else catch your eye?

**Comment by Guad** [comment]
everything else is good

**Comment by RahulTeammate** [comment]
All done.

**Comment by Guad** [comment]
Thanks for contributing

**Comment by RahulTeammate** [comment]
No problem. @Guad 
However, what will we do with listener, connection, and playerClient? If these variables will remain static, then if the user opens up 2 SM64O forms, we will have race conditions. But if we make them non-static, then if the user opens up 2 forms, they won't rewrite each other's variables.

At least, this is my rudimentary understanding of Form1.cs.

**Comment by Guad** [comment]
He can't open 2 forms. Unless you're referring to opening 2 instances of the program, in which case that wont happen.

**Comment by RahulTeammate** [comment]
I meant opening up 2 instances of form1, yes.

**Comment by wu4** [comment]
From a quick skim, I *think* this pull request removed the command-line options feature from ff5204610d8c6ccf7950d89f33d329e13ad9e6ca.

**Pull request merged**